fan-xiaomi.js

```diff
@@ -396,6 +400,7 @@ class FanXiaomi extends HTMLElement {
                 card.querySelector('.dialog').style.display = 'block'
             }*/
             this.card = card;
+            this.innerHTML = '';
```

Comment:
Would you mind explaining what this line does?

Reply:
The appendChild call on the line below will insert new html for the card content at the end of the card container. This line clears the card container contents, otherwise we end up with two copies of the card content.

We could check for existing children and do replaceChild, or there's probably more options, but just clearing the card content fist seemed like the easiest way to get to a reset state before appending.

Reply:
I see, yeah makes sense that this is the most straightforward way and there's no need to over-engineer this logic.